themes/default/content/docs/using-pulumi/adopting-pulumi/converters.md

```diff
@@ -1,6 +1,6 @@
 ---
 title_tag: "Convert Code to Pulumi"
-meta_desc: Pulumi converters allow you to convert ARM, CloudFormation, Kubernetes CustomResources, Kubernetes YAML, and Terraform to Pulumi.
+meta_desc: Pulumi convert allows you to convert ARM, CloudFormation, Kubernetes CustomResources, Kubernetes YAML, and Terraform to Pulumi.
```

Comment:
Nit, I think: I'm not sure that this is more accurate. The converters are what do the work, and this is not the reference for the `pulumi convert` command.